tsfc/finatinterface.py

```diff
@@ -70,6 +68,10 @@
     "Real": finat.DiscontinuousLagrange,
     "DPC": finat.DPC,
     "S": finat.Serendipity,
+    "SminusF": finat.TrimmedSerendipityFace,
+    "SminusDiv": finat.TrimmedSerendipityDiv,
+    "SminusE": finat.TrimmedSerendipityEdge,
+    "SminusCurl": finat.TrimmedSerendipityCurl,
```

Comment:
I think I'm only expecting to see these four added lines for this PR, do you agree?

Reply:
Yes, correct (in the end)